bindings/java/src/main/native/Makefile.am

```diff
@@ -91,13 +99,21 @@ clean-local:
 set-version:
 	$(MVNCMD) versions:set -DnewVersion=${JUCX_VERSION}
 
+# Add multi-platform support (CI only)
+repack-jar: $(jarfile)
+	unzip -o $(jarfile) -d $(java_build_dir)/repack_dir
+	\cp $(topdir)/libjucx_*.so/libjucx_*.so $(java_build_dir)/repack_dir
+	jar -cf $(jarfile) -C $(java_build_dir)/repack_dir .
+	jar tf $(jarfile)
```

Comment:
can we test this step in CI, even if we don't publish?
for example, run another step in CI just for testing the resulting JAR contains the SO of amd64 and aarch64